spire-flex/values.schema.json

```diff
@@ -83,30 +116,83 @@
                         "livePath": "/live",
                         "readyPath": "/ready"
                     }]
+                },
+                "image": {
+                    "type": "object",
+                    "default": {},
+                    "title": "The .agent.healthCheck Schema",
```

Comment:
Title wrong here?

Reply:
Nice find.  I guess I got too happy with the cut-and-paste.